src/events/event-data.types.ts

```diff
@@ -1,94 +1,24 @@
-import type {
-  APIGatewayProxyEvent,
-  APIGatewayProxyEventV2,
-  AppSyncResolverEventHeaders,
-  DynamoDBRecord,
-  KinesisStreamRecord,
-  KinesisStreamRecordPayload,
-  S3EventRecord,
-  SNSMessage,
-  SQSRecord,
-} from 'aws-lambda';
 import type { EventTrigger } from './event-trigger.enum';
 
 export type IncomingEvent = Record<string, any>;
 
-export type IncomingEventRecord = IncomingEvent['Records'][0];
-
-export type EventInfo = EventData & {
-  triggeredBy: EventTrigger;
+export type EventInfo = {
+  trigger: Trigger[];
 };
 
-export type EventData =
```

Comment:
flatten the different types and created one type the is parallel to the python_tracer and the BE